packages/access-policy/src/index.js

```diff
@@ -116,6 +116,8 @@ export class AccessPolicy {
    * @returns entities[] The entity objects
    */
   getEntitiesAllowed(permissionGroup) {
+    const allEntityCodes = Object.keys(this.policy);
+    if (!permissionGroup) return allEntityCodes;
     return Object.keys(this.policy).filter(e => this.allows(e, permissionGroup));
```

Comment:
Why run Object.keys twice?

```suggestion
    return allEntityCodes.filter(e => this.allows(e, permissionGroup));
```

Reply:
Good catch - that was the whole point of moving it out into a variable 🤦